src/bpftrace.cpp

```diff
@@ -1754,12 +1754,13 @@ std::string BPFtrace::resolve_timestamp(uint32_t mode,
   snprintf(usecs_buf, sizeof(usecs_buf), "%06" PRIu64, us);
   auto fmt = std::regex_replace(raw_fmt, usec_regex, usecs_buf);
 
-  char timestr[config_.get(ConfigKeyInt::max_strlen)];
-  if (strftime(timestr, sizeof(timestr), fmt.c_str(), &tmp) == 0) {
+  uint64_t timestr_size = config_.get(ConfigKeyInt::max_strlen);
+  std::vector<char> timestr(timestr_size);
```

Comment:
```suggestion
  std::string timestr(timestr_size, '\0');
```
Same. If we create it as a string then we don't have to copy it at the end